aces/Loris
# [DQT] 'Download Table as CSV' broken if StaticDataTable has non-empty data from mri_data field

## Brief summary of changes

Here is a bugfix PR for an issue with the CSV download in the DQT module. The StaticDataTable.js has `data` containing `react link elements` from the DQT module and will result in the an error being thrown. I'm still investigating if a better solution exists by modifying the DQT instead of cleaning the data in the StaticDataTable.js and before it's parsed to a worker for CSV download.

#### Testing instructions (if applicable)

0. If you don't already have mri data in the DQT, run `php tools/CouchDB_Import_MRI.php`
1. Go to 'Data Query Tool' module
2. Click on 'Define Fields' tab, select 'mri_data' in 'Instrument:' field, input 'Selected_dwi25' in 'Search within instrument:' field.
3. Select the 'Selected_dwi25' div that shows up below
4. Go to 'View Data' tab, click on 'Run Query'.
5. Once the data table populates, scroll down to the bottom of the page, and click on 'Download Table as CSV' button.
6. Notice the CSV now downloads (before PR there would be an error)

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
6425
